services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -83,6 +80,28 @@ export const ChoiceListField = ({
     });
   };
 
+  const clearNestedValues = (choices: FieldChoice[]) => {
+    return choices.forEach((choice: FieldChoice) => {
```

Comment:
Suggestion: Because we aren't using the output value anywhere, there's no real need to return anything here.